app/views/spotlight/appearances/edit.html.erb

```diff
@@ -1,8 +1,9 @@
+<% jcrop_options = {croppable: true, selector: 'masthead_image', bg_color: 'black', bg_opacity: '.4', max_size: '[1800,131]', min_size: '[1200,120]'} %>
```

Comment:
Yeah, I think I'll push it into a helper.  Some of the stuff doesn't make sense to be configured in the Engine config (like selector/croppable).

I'll probably do some experimenting w/ the Jcrop stuff and the masthead size/proportions once this is merged in, so may end up doing some more stuff w/ this config that could end up lending itself to an Engine configuration.